Lesson6/6_Args_sum.py

```diff
@@ -0,0 +1,35 @@
+'''
+Args sum - write a function called sum_all
+that takes any number of arguments and returns their sum.
+'''
+
+
+def sum_all(*args):
```

Comment:
najprostsze rozwiązanie to:
```
def sum_all(*args):
    return sum(args)
```
a takie, które obsłuży też to o czym piszę poniżej to:
```
def sum_all(*args):
  return sum(args[1:], args[0])
```